src/components/document-metadata-modal.yml

```diff
@@ -32,3 +35,8 @@ sequenceInfo:
   de: |
     Du kannst mehrere Dokumente gleichzeitig anlegen.
     An den Titel bzw. URL-Pfad wird für jedes Dokument eine laufende Nummer angehängt.
+verifiedInfo:
+  en:
+    As a content maintainer, you can vouche for the quality of this document. Verified documents have higher relevance in the search results.
+  de:
+    Als Inhalt-Maintainer kannst du die Qualität <something something>. Überprüfte Dokumente haben eine höchere Relevanz auf der Suchseite.
```

Comment:
As first sentence maybe: "Als Maintainer kannst du bestätigen, dass das Dokument den Qualitätsanforderungen entspricht."

In the second sentence it should be "hö**h**ere".

Reply:
Sehr schön, danke!